packages/circleloader/src/spectrum-circleloader.css

```diff
@@ -10,6 +10,378 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 
 THIS FILE IS MACHINE GENERATED. DO NOT EDIT */
+@keyframes spectrum-fill-mask-1 {
```

Comment:
Now that the processor pulls this in we are able to remove this code from the manually built file above.